spark-jobs/src/main/scala/za/co/absa/enceladus/conformance/HyperConformance.scala

```diff
@@ -78,11 +78,13 @@ class HyperConformance (menasBaseUrls: List[String],
     // using HDFS implementation until HyperConformance is S3-ready
     implicit val hdfs: FileSystem = FileSystem.get(sparkSession.sparkContext.hadoopConfiguration)
     implicit val hdfsUtils: HadoopFsUtils = HadoopFsUtils.getOrCreate(hdfs)
+    val function: (DataFrame, String) => DataFrame = (df: DataFrame, _) =>
```

Comment:
that non-descriptive name of function-val you already fixed elsewhere -- here, you might have forgotten to change it, too.

Reply:
I noticed a mistake. It's not only the naming that I did wrong here